### PR TITLE
fix:issues

### PR DESCRIPTION
**Pull request by MayueCif**

### CODE REVIEW

File: src/Services/Masa.Auth.Service.Admin/Services/MessageService.cs [hunk 1]
```diff
@@ -10,14 +10,14 @@ public MessageService() : base("api/message")
         RouteOptions.DisableAutoMapRoute = false;
     }
 
-    [AllowAnonymous]
+    [ProductionDisabledRoute]
```
Comment (doddgu) [inline src/Services/Masa.Auth.Service.Admin/Services/MessageService.cs:13]: ProductionDisabledRoute?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor [hunk 2]
```diff
@@ -6,7 +6,8 @@
             <MForm Model=_menuPermissionDetailDto EnableValidation @ref="_form">
                 <MRow>
                     <MCol Md=12>
-                        <STextField Label="@T("Name")" Required @bind-Value="_menuPermissionDetailDto.Name" />
+                        @*<STextField Label="@T("Name")" Required @bind-Value="_menuPermissionDetailDto.Name" />*@
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/AddMenuPermission.razor:9]: ?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/DccI18nConfiger.razor [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    [Parameter]
+    public EventCallback<string> NameChanged { get; set; }
+
+    bool _dialog;
+
+    private Task OpenConfigDialog()
+    {
+        var publicSection = ConfigurationApi.GetPublic();
+        publicSection.GetChildren();
+        //todo
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Pages/RolePermissions/Permissions/DccI18nConfiger.razor:55]: What todo?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Shared/AdminCompontentBase.cs [hunk 4]
```diff
@@ -81,11 +84,16 @@ public string T(string key)
         else return I18n.T(key, true);
     }
 
-    public string T(string formatkey,params string[] args)
+    public string T(string formatkey, params string[] args)
     {
         return string.Format(T(formatkey), args);
     }
 
+    protected string DT(string key)
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Shared/AdminCompontentBase.cs:92]: What's the difference between DT and T?

File: src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+@inject IWebHostEnvironment _environment
+@layout SimpleLayout
+
+<MCard Width="1000" Style="height: calc(100vh - 200px) !important;">
+    <MCardTitle Class="justify-center">
+        MASA Stack用户协议
+    </MCardTitle>
+    <MCardText Style="overflow-y: scroll;">
+        1. 特别提示
+        XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor:12]: Use DCC

File: src/Web/Masa.Auth.Web.Sso/Program.cs [hunk 6]
```diff
@@ -73,6 +73,7 @@
 
 builder.Services.AddSingleton<SsoAuthenticationStateCache>();
 builder.Services.AddScoped<AuthenticationStateProvider, SsoAuthenticationStateProvider>();
+//builder.Services.AddTransientDecorator<IAuthenticationSchemeProvider, DynamicAuthenticationSchemeProvider>();
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Sso/Program.cs:76]: ?

File: src/Web/Masa.Auth.Web.Sso/wwwroot/i18n/en-US.json [hunk 7]
```diff
@@ -1,12 +1,12 @@
 {
   "$Text": "English",
   "Cancel": "Cancel",
-  "UserName": "User Name",
-  "UserNamePlaceHolder": "Place Enter User Name",
+  "Account": "Acount",
+  "AccountPlaceHolder": "Place Enter Account/PhoneNumber/Email",
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Sso/wwwroot/i18n/en-US.json:5]: PhoneNumber -> Phone Number

**Review by doddgu — CHANGES_REQUESTED** [review]

File: Masa.Auth.sln [hunk 8]
```diff
@@ -46,6 +46,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Auth.ApiGateways.Calle
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Auth.Security.OAuth.Providers", "src\Web\Masa.Auth.Security.OAuth.Providers\Masa.Auth.Security.OAuth.Providers.csproj", "{D58D018F-26B0-45D1-AD0B-E2777C59ED44}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Stack.Components", "..\MASA.Stack.Components\src\Masa.Stack.Components\Masa.Stack.Components.csproj", "{06178C79-5EC9-426C-A2C8-759424775AF9}"
```
Comment (doddgu) [inline Masa.Auth.sln:49]: ?

File: src/Web/Masa.Auth.Web.Admin.Rcl/Masa.Auth.Web.Admin.Rcl.csproj [hunk 9]
```diff
@@ -19,14 +19,15 @@
 		<PackageReference Include="Humanizer.Core" Version="2.14.1" />
 		<PackageReference Include="Humanizer.Core.zh-CN" Version="2.14.1" />
 		<PackageReference Include="Mapster" Version="7.3.0" />
-		<PackageReference Include="Masa.Stack.Components" Version="0.0.168" />
+		<!--<PackageReference Include="Masa.Stack.Components" Version="0.0.168" />-->
```
Comment (doddgu) [inline src/Web/Masa.Auth.Web.Admin.Rcl/Masa.Auth.Web.Admin.Rcl.csproj:22]: ?

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Services/Masa.Auth.Service.Admin/Services/MessageService.cs (hunk 1 above)
Comment (MayueCif) [reply src/Services/Masa.Auth.Service.Admin/Services/MessageService.cs:13]: demo environment diabled operate

File: src/Web/Masa.Auth.Web.Admin.Rcl/Masa.Auth.Web.Admin.Rcl.csproj (hunk 9 above)
Comment (MayueCif) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Masa.Auth.Web.Admin.Rcl.csproj:22]: draft，Not ready yet... wait

File: src/Web/Masa.Auth.Web.Admin.Rcl/Shared/AdminCompontentBase.cs (hunk 4 above)
Comment (MayueCif) [reply src/Web/Masa.Auth.Web.Admin.Rcl/Shared/AdminCompontentBase.cs:92]: T get value from i18n file,DT get value from Dcc

File: src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor (hunk 5 above)
Comment (MayueCif) [reply src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor:12]: should sso client custom login setting,first provider page,next version improve function

Comment (doddgu) [reply src/Web/Masa.Auth.Web.Sso/Pages/Protocol.razor:12]: ok

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**